.github/workflows/ci.yml

```diff
@@ -18,10 +21,59 @@ jobs:
           ruby-version: .ruby-version
           bundler-cache: true
 
-      - name: Install dependencies
-        run: |
-          bundle install
```

Comment:
we don't we do this anymore?

Reply:
[Linking the docs](https://github.com/ruby/setup-ruby?tab=readme-ov-file#single-job) for posterity, but it looks like that is indeed the case:
`bundler-cache: true # runs 'bundle install' and caches installed gems automatically`